sky/provision/gcp/instance_utils.py

```diff
@@ -285,13 +284,13 @@ def create_node_tag(cls,
                         is_head: bool = True) -> str:
         if is_head:
             node_tag = {
-                TAG_SKYPILOT_HEAD_NODE: '1',
-                TAG_RAY_NODE_KIND: 'head',
+                provision_constants.TAG_SKYPILOT_HEAD_NODE: '1',
+                provision_constants.TAG_RAY_NODE_KIND: 'head',
             }
         else:
             node_tag = {
-                TAG_SKYPILOT_HEAD_NODE: '0',
-                TAG_RAY_NODE_KIND: 'worker',
+                provision_constants.TAG_SKYPILOT_HEAD_NODE: '0',
+                provision_constants.TAG_RAY_NODE_KIND: 'worker',
             }
```

Comment:
Should we move this function to `sky/provision/common.py`? Seems like it is reused a lot of times